andela/pippin-ah-frontend
# [Feature #164102493] Fix required signin refresh

#### What does this PR do?
- It ensures a user does not have to refresh to signin after signing out

#### Description of Task to be completed?
- [x] fix signin refresh error

#### How should this be manually tested?
- signout from the app and click on signin again
- it should navigate to signin page

#### What are the relevant pivotal tracker stories?
- [#164102493](https://www.pivotaltracker.com/story/show/164102493)
